packages/bridge-ui/src/components/BridgeForm/BridgeForm.svelte

```diff
@@ -357,7 +357,7 @@
       // Set it manually.
       tx.chainId = $srcChain.id;
 
-      const userAddress = await $signer.getAddress();
+      const userAddress = (await $signer.getAddress()) as Address;
```

Comment:
```suggestion
      const userAddress: Address = (await $signer.getAddress());
```
nit

Reply:
can we use this? it's safer if possible (check at compile time instead of runtime error)

Reply:
That doesn't work. You cannot assign a `string` to an `Address`. You need to cast the `string`